crates/emitter/src/emitter.rs

```diff
@@ -429,14 +404,14 @@ impl InstructionWriter {
     }
 
     pub fn patch_jump_to_target(&mut self, target: BytecodeOffset, jump: BytecodeOffset) {
-        let diff = BytecodeOffsetDiff::new(target, jump).diff as i32;
+        let diff = i32::from(target.diff_from(jump));
```

Comment:
I wonder if `let diff = target.diff_from(jump).into();` works here, given `LittleEndian::write_i32(..., diff)` implies `diff` is `i32`